pkg/ocm/accounts_mgmt.go

```diff
@@ -0,0 +1,51 @@
+package ocm
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/go-openapi/strfmt"
+	amgmtv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+	"github.com/openshift/assisted-service/pkg/commonutils"
+
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/pkg/errors"
+)
+
+//go:generate mockgen -package ocm -destination mock_accounts_mgmt.go . OCMAccountsMgmt
+type OCMAccountsMgmt interface {
+	CreateSubscription(ctx context.Context, clusterID strfmt.UUID) (*amgmtv1.Subscription, error)
+}
+
+type accountsMgmt struct {
+	client *Client
+}
+
+func (a accountsMgmt) CreateSubscription(ctx context.Context, clusterID strfmt.UUID) (*amgmtv1.Subscription, error) {
```

Comment:
where are the tests?

Reply:
Not written yet, I will write them once I have the ability to test it manually (need `ocm` permissions and a new ID for `AssistedInstall` clusters in AMS).

Reply:
Done with wiremock